src/states/bootstrapping.rs

```diff
@@ -49,15 +54,15 @@ pub enum TargetState {
 
 // State of Client or Node while bootstrapping.
 pub struct Bootstrapping {
-    pub(super) action_sender: RoutingActionSender,
-    pub(super) cache: Box<Cache>,
-    pub(super) crust_service: Service,
-    pub(super) full_id: FullId,
-    pub(super) min_section_size: usize,
-    pub(super) timer: Timer,
+    action_sender: RoutingActionSender,
```

Comment:
Wouldn't it make sense to just use composition to avoid repeating all these fields?